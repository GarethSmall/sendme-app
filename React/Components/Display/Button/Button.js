// @flow
import React, { PureComponent } from 'react';
import ReactNative from 'react-native';
import AppTheme from '../../../AppTheme/AppTheme';

export default class Button extends PureComponent {
  render() {
    const { ...other } = this.props;
    return <ReactNative.Button {...other} color={AppTheme.secondaryColor} />;
  }
}
