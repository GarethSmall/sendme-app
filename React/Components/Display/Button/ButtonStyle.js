import { StyleSheet } from 'react-native';
import AppTheme from '../../../AppTheme/AppTheme';

export default ButtonStyle = StyleSheet.create({
  input : {
    height: 45,
    backgroundColor: AppTheme.secondaryColor,
    borderRadius: AppTheme.smallSize,
  },
});