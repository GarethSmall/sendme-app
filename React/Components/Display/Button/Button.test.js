// @flow
import React from 'react';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import shallow from 'enzyme/shallow';
import Button from './';
const { describe, test, expect } = global;

Enzyme.configure({ adapter: new Adapter() });

describe('Button ', () => {
  test('should render', () => {
    const component = shallow(<Button title="Test Button" onPress={() => {}} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
