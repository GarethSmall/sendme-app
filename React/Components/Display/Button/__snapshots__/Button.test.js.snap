// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button  should render 1`] = `
<Button
  color="#93748a"
  onPress={[Function]}
  title="Test Button"
/>
`;
